frontends/web/src/routes/account/summary/chart.tsx

```diff
@@ -101,10 +101,11 @@ export const Chart = ({
   const ref = useRef<HTMLDivElement>(null);
   const refToolTip = useRef<HTMLSpanElement>(null);
   const chart = useRef<IChartApi>();
+  const chartInitialized = useRef(false);
```

Comment:
I guess we could also use `chart.current` but I like it like that, it is more clear even if its a bit redundant. but no strong opinion, let me know if I should remove this and use `chart.current` (!== undefined) to check if the chart was initialized.